.eslintrc

```
@@ -0,0 +1,12 @@
+{
+    "extends": "eslint-config-neo/config-backend",
+    "rules": {
+        "no-console": "off",
+        "import/no-extraneous-dependencies": [
+            "error",
+            {
+                "devDependencies": true
+            }
+        ]
+    }
+}
```

someone may ask you to add a newline at the end of this file .... 
```suggestion
}

```